### PR TITLE
remove PETHContractWrapper

### PR DESCRIPTION
**Pull request by daiki44**
## What?
WAK-348

- remove auto wrap function from PETH (remove PETHContractWrapper)
- remove registerCustomToken method

## Integration test result
https://github.com/cryptoeconomicslab/gazelle/runs/779102530?check_suite_focus=true

### CODE REVIEW

**Comment by daiki44** [comment]
I have to update the integration test.

File: packages/plasma-light-client/src/LightClient.ts [hunk 1]
```diff
@@ -201,6 +201,10 @@ export default class LightClient {
     )
   }
 
+  public getWallet(): Wallet {
```
Comment (daiki44) [inline packages/plasma-light-client/src/LightClient.ts:204]: What do you think about this method?

Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts:204]: why do you need this?

Comment (daiki44) [reply packages/plasma-light-client/src/LightClient.ts:204]: I'm using this method here.
https://github.com/cryptoeconomicslab/gazelle/blob/d61b62520d050f84315c2487e2b026b1c2e0cd62/integration-test/e2e-tests/__tests__/index.test.ts#L97

Comment (daiki44) [reply packages/plasma-light-client/src/LightClient.ts]: removed!
https://github.com/cryptoeconomicslab/gazelle/pull/331/commits/06ef5f616fb2844611400ca2bb32a7c37f7b019f

Comment (tkmct) [reply packages/plasma-light-client/src/LightClient.ts]: you can just make wallet field `readonly`.

**Pull request merged**